Author: Andreas Tille <tille@debian.org>
Last-Update: Fri, 17 Mar 2017 10:23:15 +010
Description: Remove duplicated definition which is provided by libsquid

--- a/algorithm-hmm/src/include/funcs.h
+++ b/algorithm-hmm/src/include/funcs.h
@@ -170,7 +170,6 @@ extern void  LogNorm(float *vec, int n);
 extern float Logp_cvec(float *cvec, int n, float *alpha);
 extern void  SampleDirichlet(float *alpha, int n, float *p);
 extern float SampleGamma(float alpha);
-extern void  SampleCountvector(float *p, int n, int c, float *cvec);
 extern float P_PvecGivenDirichlet(float *p, int n, float *alpha);
 
 /* from misc.c
